lib/metrics/context.js

```diff
@@ -19,7 +19,12 @@ const SCHEMA = isA.object({
   // only Sync creates records in the devices table.
   deviceId: isA.string().length(32).regex(HEX_STRING).optional(),
   flowId: isA.string().length(64).regex(HEX_STRING).optional(),
-  flowBeginTime: isA.number().integer().positive().optional()
+  flowBeginTime: isA.number().integer().positive().optional(),
+  utmCampaign: isA.string().optional(),
+  utmContent: isA.string().optional(),
+  utmMedium: isA.string().optional(),
+  utmSource: isA.string().optional(),
+  utmTerm: isA.string().optional()
```

Comment:
At first glance, you might be thinking:

> These properties should be validated!

I don't think they should.

If we add strict validation, it means an error on the marketing side could prevent people from creating accounts or signing in. That would be bad.

And... since all we're doing is passing these values out of the server, it's not like validation would protect us from anything in this repo. The layer(s) that interpret these values are welcome to validate them if they wish.

And... and... (warning: logical fallacy incoming!) [we never used to validate these properties before](https://github.com/mozilla/fxa-auth-server/pull/1585/files#diff-3833ef223ad2c72e2c7db2440f69353eL26), so it must be fine.

Reply:
I typically like being more strict in what we accept, but this sounds reasonable.